src/main/java/com/cloudogu/smeagol/wiki/domain/WikiId.java

```diff
@@ -0,0 +1,45 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import java.util.Objects;
+
+/**
+ * Wiki identifier, a wiki can be identified by the id of the repository and its branch.
+ */
+public class WikiId {
+
+    // TODO should be name and repositorID ???
+    private final String repositoryID;
+    private final String branch;
+
+    public WikiId(String repositoryID, String branch) {
+        this.repositoryID = repositoryID;
+        this.branch = branch;
+    }
+
+    public String getRepositoryID() {
+        return repositoryID;
+    }
+
+    public String getBranch() {
+        return branch;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)